common/src/test/java/org/opensearch/ml/common/connector/HttpConnectorTest.java

```diff
@@ -189,6 +189,15 @@ public void createPayload() {
         Assert.assertEquals("{\"input\": \"test input value\"}", predictPayload);
     }
 
+    @Test
+    public void createPayload_Skip_Parameter_Validation() {
+        exceptionRule.expect(IllegalArgumentException.class);
+        exceptionRule.expectMessage("Some parameter placeholder not filled in payload: input");
```

Comment:
Why we expect to see this error with `skip_validating_missing_parameters` as `true`

Reply:
Added new UT in ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/RemoteConnectorExecutor.java